src/app/advertisement/advertisement.component.ts

```diff
@@ -28,6 +28,8 @@ export class AdvertisementComponent implements OnInit {
           this.getAdvertisements(this.advertisementService.getAllAdvertisements);
         } else {
           this.getAdvertisements(this.searchAdvertisementService.searchAdvertisementByTitle, queryParam.title);
+          this.getAdvertisements(this.searchAdvertisementService.searchAdvertisementByTag, queryParam.tag);
```

Comment:
I see your point, I will solve what you said.
I'm going to solve that problem during the next sprint .
Thanks for reviewing.